.gitmodules

```diff
@@ -35,4 +34,10 @@
 	url = https://github.com/taocpp/PEGTL.git
 [submodule "tools/meta/ext/inja"]
 	path = tools/meta/ext/inja
-url=https://github.com/pantor/inja.git
\ No newline at end of file
+	url=https://github.com/pantor/inja.git
+[submodule "modules/cimg/ext/openexr/openexr"]
+	path = modules/cimg/ext/openexr/openexr
+	url = https://petersteneteg@github.com/inviwo/openexr.git
```

Comment:
you cloned into a private repository?

Reply:
hmm the name should not have to be there, the repo should be public
will fix